package.json

```diff
@@ -14,45 +14,46 @@
     "dist"
   ],
   "devDependencies": {
-    "@babel/cli": "^7.1.2",
-    "@babel/core": "~7.3.0",
-    "@babel/plugin-transform-runtime": "~7.3.0",
-    "@babel/preset-env": "~7.3.0",
-    "@babel/runtime-corejs2": "~7.3.0",
-    "babel-loader": "v8.0.0-beta.0",
+    "@babel/cli": "^7.4.4",
+    "@babel/core": "~7.4.4",
```

Comment:
@fgutmann Hey! Sorry for disturbing. As I remember we have a few problems with upgrading babel/core after version 7.4. Do you remember what kind of problems? Thanks. 🙂 It was some of your PR.

Reply:
Hi! The issue was that babel-preset-env had a transitive dependency to core-js 3 introduced. This has an incompatible directory structure with core-js 2 and broke our build. Babel 7.4 is officially compatible with core js 2 though. A workaround back then was to add an explicit dependency to core-js 2.

I guess that the dependency from babel-preset-env to core-js 3 was a bug on the babel side. The dependencies of that package seem to have changed in the meantime. Back then it was very early days for babel 7.4. So all of that might not be an issue anymore.

See here for reference of the old issue: https://github.com/JetBrains/kotlin-playground/pull/48#discussion_r268667184

Reply:
Thanks a lot 🙂